libgadget/timebinmgr.h

```diff
@@ -60,7 +61,7 @@ inttime_t out_from_ti(inttime_t ti);
 
 int OutputListAction(ParameterSet * ps, char * name, void * data);
 void set_sync_params(int OutputListLength, double * OutputListTimes);
-void setup_sync_points(double TimeIC, double TimeMax, double no_snapshot_until_time, int SnapshotWithFOF);
+void setup_sync_points(double TimeIC, double TimeMax, double UVBGTimestep, double no_snapshot_until_time, int SnapshotWithFOF);
```

Comment:
I thought about not using the syncpoints here, and calculating the UVBG if it has been longer than UVBGTimestep since the last calculation, by looking at the time difference between steps. Not quite sure how the cadence would work out in that case though.

Reply:
I actually really like your use of extra syncpoints, it's a neat application. Looking at the time difference is clever too, but I like this more.